app/Models/Collection.php

```diff
@@ -24,6 +24,13 @@ public function collectionType()
     return $this->belongsTo(CollectionType::class);
   }
 
+  public function collectionTypeName()
```

Comment:
Why repeat this method in Collection and Prefix rather than making it an instance method of CollectionType?

Reply:
Oh, huh. I guess I didn't think about it because it's called in each object's view. 
`{{ $record->collectionTypeName() }}` 
I'll work on DRYing that up.